src/ekat/kokkos/ekat_kokkos_utils.hpp

```diff
@@ -119,22 +103,27 @@ struct ExeSpaceUtils<Kokkos::Cuda> {
 #endif
 
 /*
- * TeamUtils contains utilities for getting concurrency info for
- * thread teams. Don't use _TeamUtilsCommonBase directly, use
- * TeamUtils.
+ * TeamUtils contains utilities for getting concurrency info for thread teams.
+ * You cannot use it directly (protected c-tor. You must use TeamUtils.
```

Comment:
close parens around "protected c-tor"

Reply:
Wow, you really are reading every line! Thanks! =)

Reply:
Skimming. :-) Just happened to scan this one.